Sources/Datadog/Core/DatadogCoreProtocol.swift

```diff
@@ -0,0 +1,54 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+
+internal var DefaultDatadogCore: DatadogCoreProtocol = NOOPDatadogCore()
+
+/// A Datadog Feature registry hold a set of features and is responsible of
+/// managing their storage and upload mechanism. It also provide thread-safe
+/// scope for features.
```

Comment:
Small typos and it's a bit outdated after last renaming, how about:
```suggestion
/// A Datadog Core holds a set of features and is responsible of
/// managing their storage and upload mechanism. It also provides a thread-safe
/// scope for writing events.
```